Modules/Segmentation/LevelSets/include/itkParallelSparseFieldLevelSetImageFilter.h

```diff
@@ -710,13 +698,8 @@ class ITK_TEMPLATE_EXPORT ParallelSparseFieldLevelSetImageFilter:
    *  volume  */
   int *m_ZCumulativeFrequency{nullptr};
 
-  /** A global barrier used for synchronization between all threads. */
-  typename Barrier::Pointer m_Barrier;
-
   /** Local data for each individual thread. */
   struct ThreadData {
-    char pad1[128];
```

Comment:
Is [false sharing ](https://en.wikipedia.org/wiki/False_sharing)no longer possible with the revised implementation?

Reply:
Here it a simple case where there was a significant multi-threading performance issue due to it:
https://github.com/InsightSoftwareConsortium/ITK/commit/ca4bd1c18d48118d1d96a22214534bee478e7de5#diff-e125cabfc7ed03d932f380888fb9846c